implementation/rest/src/main/java/io/smallrye/opentelemetry/implementation/rest/OpenTelemetryServerFilter.java

```diff
@@ -64,7 +64,9 @@ public OpenTelemetryServerFilter(final OpenTelemetry openTelemetry) {
     @Override
     public void filter(final ContainerRequestContext request) {
         Context parentContext = Context.current();
-        if (instrumenter.shouldStart(parentContext, request)) {
+        // In scenarios where this library is installed but CDI is not enabled, instrumenter will be null. In thsese
```

Comment:
Maybe just a single null check for the entire method? Like we have in the ClientFilter

Reply:
Works for me. Updated.